packages/building-blocks/src/Alert.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { ReactChild, useState } from "react";
+import styled from "styled-components";
+import AnimateHeight from "react-animate-height";
+import { ChevronUp } from "@styled-icons/bootstrap/ChevronUp";
+import { Bell } from "@styled-icons/bootstrap/Bell";
+import blue from "./colors/blue";
+
+interface Props {
+  alertHeader: string;
+  alertSubheader?: string;
+  backgroundColor?: string;
+  collapsable?: boolean;
+  children?: ReactChild;
+  Icon?: any;
+}
+
+const AlertContainer = styled.div<{
```

Comment:
Should we consider going more semantic and using a [details](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) tag?

Reply:
Hmm I think this would actually make the component less customizable, since `details` always has a toggle, and this alert currently only has a toggle if `collapsible` is true

Reply:
is it possible to hide the toggle with CSS?